www/src/pages/en/usage/trpc.md

```diff
@@ -32,6 +32,25 @@ tRPC allows us to write end to end, typesafe APIs without any code generation or
   </cite>
 </blockquote>
 
+## Files
+
+tRPC requires quite a lot of boilerplate that `create-t3-app` sets up for you. Let's go over the files that are generated for you:
```

Comment:
tRPC requires quite a lot of boilerplate that `create-t3-app` sets up for you. Let's go over the files that are generated: